src/app.rs

```diff
@@ -566,25 +519,16 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
         }))
     }
 
-    #[deprecated(
-        since = "0.5.0",
-        note = "Use `builder` with `AppBuilder::{after_mount, before_mount}` instead."
-    )]
-    pub fn build(
-        init: impl FnOnce(Url, &mut OrdersContainer<Ms, Mdl, INodes, GMs>) -> Init<Mdl> + 'static,
-        update: UpdateFn<Ms, Mdl, INodes, GMs>,
-        view: ViewFn<Mdl, INodes>,
-    ) -> InitAppBuilder<Ms, Mdl, INodes, GMs> {
-        Self::builder(update, view).init(Box::new(init))
-    }
-
     /// App initialization: Collect its fundamental components, setup, and perform
     /// an initial render.
+    /// TODO: the `builder` module has been removed from the public interface, but
+    /// weaves a complex web of dependencies. This function is also a dependency
+    /// to [`App::start`] and other functions.
     #[deprecated(
         since = "0.4.2",
-        note = "Please use `AppBuilder.build_and_start` instead"
+        note = "Legacy note: Please use `AppBuilder.build_and_start` instead\nAs part of the 0.7.0 cleanup, this has been made private. `App::start` depends on this function."
```

Comment:
I think that `run` will be also removed - don't kill time with writing nice docs for it :)